.eslintrc.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     'react-hooks',
     'react-native',
     'sort-keys-fix',
+    'sort-destructure-keys',
```

Comment:
could if we're sorting the plugins alphabetically, then this would need to be moved one line up